sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentSentiment.cs

```diff
@@ -14,11 +14,11 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public class DocumentSentiment
     {
-        internal DocumentSentiment(TextSentimentLabel sentiment, double positiveScore, double neutralScore, double negativeScore, List<TextSentiment> sentenceSentiments)
+        internal DocumentSentiment(TextSentimentLabel sentiment, double positiveScore, double neutralScore, double negativeScore, List<SenteceSentiment> sentenceSentiments)
```

Comment:
It feels inconsistent for some model types to be named as **Text**SentimentLabel and others be **Sentence**Sentiment?

Reply:
ugh, yeah. that `Text` thing doesn't work. 
I am going to check wherever we have that

Reply:
so, there are other types that have `Text` in it. but it will involve a bigger change.
So for now, I am going to change `TextSentimentLabel` to `SentimentLabel`
after p2, we can look at https://github.com/Azure/azure-sdk-for-net/issues/8900 to see where we should keep/remove the words document and text